packages/spectacular/src/lib/feature-testing/feature-testing-module/spectacular-feature-testing.module.ts

```diff
@@ -0,0 +1,52 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { featurePathToken } from '../configuration/feature-path.token';
+import { SpectacularFeatureTestingModuleOptions } from './spectacular-feature-testing-module-options';
+import { SpectacularFeatureTestingRootModule } from './spectacular-feature-testing-root.module';
+
+/**
+ * Configures the `RouterTestingModule` and provides Spectactular
+ * services for testing feature modules
+ *
+ * NOTE! Do not import directly. Use `SpectacularFeatureTestingModule.withFeature`.
+ *
+ * NOTE! Prefer to use `SpectacularFeatureTestbed`. This Angular module is a low
+ * level building block in case you need more control over your test setup.
+ */
+@NgModule()
+export class SpectacularFeatureTestingModule {
+  /**
+   * Configures the `RouterTestingModule` and provides Spectactular
+   * services for testing feature modules
+   */
+  static withFeature<TFeatureModule>({
+    featureModule,
```

Comment:
What's your suggestion for `SpectacularCreateFeatureOptions`?

Reply:
`path` and `module` are not very clear names in my opinion.

Reply:
The same applies to all modules, etc where `featureModule` and `featurePath` is used.
>What's your suggestion for SpectacularCreateFeatureOptions (now SpectacularCreateFeatureHarnessOptions).

Reply:
The name of the module of indicates it is in a feature context thats why am saying make no sense :)
>path and module are too generic and ambivalent names in my opinion.

Reply:
I don't like the more generic, shorter forms at all. At best, it will save a few strokes on your keyboard (not really because of IntelliSense). At worst, it will cause confusion and mistakes.

Reply:
Do you realize how many types of Angular modules there are in Angular? We're looking for a feature module. That we are creating a feature harness doesn't necessarily spell out that we're looking for a feature module. It could be the app module, or a router module. However, that's not the case. Path for what? Route? File? Namespace? No, it's the feature route path. Maybe we should make this even more explicit.